include/zephyr/net/wifi_mgmt.h

```diff
@@ -723,6 +723,10 @@ struct wifi_connect_req_params {
 	uint8_t ignore_broadcast_ssid;
 	/** Parameter used for frequency band */
 	enum wifi_frequency_bandwidths bandwidth;
+	/** Domain suffix match */
+	const uint8_t *domain_suffix_match;
```

Comment:
Please add helpful text and also wpa_supplicant supports passing multiple names too, we should mention that too.

Reply:
also maybe this should be a user friendly name not wpa_supplicant internal name, e.g., 
```suggestion
    /** Domain name in server certificate will be verified if this is provided */
	const uint8_t *server_cert_domain;
```